views/components/PartnerApp.js

```diff
@@ -4,11 +4,13 @@ import SignUpForm from './SignUpForm';
 
 class PartnerApp extends React.Component {
   render() {
-    const { name, imageUrl, copy, partnerId } = this.props;
+    const { name, imageUrl, copy, partnerId, campaignKeyBeta, additionalFields } = this.props;
     const formDetails = {
       header: `${name}`,
       subhead: `${copy}`,
       partnerId: partnerId,
+      betaOptInPath: campaignKeyBeta,
+      extras: additionalFields,
```

Comment:
Yea that was a consideration. I ended up not touching it since `partnerId` was still getting used in a couple places and I wasn't using `hideAlpha` with the work I was doing. It does probably impact what you're doing though. If you're working off of changes from this branch, you're welcome to make whatever edits work best with the BTS build.